Tools/LyTestTools/ly_test_tools/_internal/pytest_plugin/editor_test.py

```diff
@@ -3,48 +3,54 @@
 
 
 def pytest_addoption(parser: argparse.ArgumentParser) -> None:
     """
-    Options when running editor tests in batches or parallel.
+    Options when running multiple tests in batches or parallel.
     :param parser: The ArgumentParser object
     :return: None
     """
-    parser.addoption("--no-editor-batch", action="store_true", help="Disable batching multiple tests within each single editor")
-    parser.addoption("--no-editor-parallel", action="store_true", help="Disable running multiple editors in parallel")
+    parser.addoption("--no-test-batch", action="store_true", help="Don't batch multiple tests in single instance")
+    parser.addoption("--no-test-parallel", action="store_true", help="Don't run multiple instances in parallel")
     parser.addoption("--editors-parallel", type=int, action="store",
-                     help="Override the number editors to run at the same time. Tests can override also this value, "
-                          f"which has a higher precedence than this option. Default value is: "
-                          f"{ly_test_tools.o3de.editor_test.EditorTestSuite.get_number_parallel_editors()}")
+                     help="Override the number of Editors to run at the same time. Default value is: "
+                          f"{ly_test_tools.o3de.multi_test_framework.AbstractTestSuite.get_number_parallel_instances()}")
+    parser.addoption("--material-editors-parallel", type=int, action="store",
+                     help="Override the number of MaterialEditors to run at the same time. Default value is "
+                          f"{ly_test_tools.o3de.multi_test_framework.AbstractTestSuite.get_number_parallel_instances()}")
 
 
-def pytest_pycollect_makeitem(collector: PyCollector, name: str, obj: object) -> PyCollector:
+def pytest_pycollect_makeitem(collector: _pytest.python.Module, name: str, obj: object) -> _pytest.python.Module:
     """
-    Create a custom custom item collection if the class defines pytest_custom_makeitem function. This is used for
+    Create a custom item collection if the class defines a pytest_multitest_makeitem method. This is used for
     automatically generating test functions with a custom collector.
+    Classes that inherit the AbstractTestSuite class require a "pytest_multitest_makeitem" method to collect tests.
     :param collector: The Pytest collector
     :param name: Name of the collector
-    :param obj: The custom collector, normally an EditorTestSuite.EditorTestClass object
+    :param obj: The custom collector, normally a test class object inside the AbstractTestSuite class
     :return: Returns the custom collector
     """
     if inspect.isclass(obj):
         for base in obj.__bases__:
-            if hasattr(base, "pytest_custom_makeitem"):
-                return base.pytest_custom_makeitem(collector, name, obj)
+            if hasattr(base, "pytest_multitest_makeitem"):
+                return base.pytest_multitest_makeitem(collector, name, obj)
```

Comment:
This provided a more generic approach where you can do more custom make items that doesn't need to specifically be multitest

Reply:
It was renamed so that it serves as a collector for `mult_test_framework.py` tests as those are the tests that use the `AbstractTestClass` for test collection. It's not super important but I feel it clarifies the usage of the function better.